playwright/test/landing-pages.test.ts

```diff
@@ -64,4 +66,22 @@ describe('Top-level landing pages', () => {
 
     expect(content).toBe('About us');
   });
+
+  test('the works page renders with a status code of 200', async () => {
```

Comment:
Do we want to make sure that we get a 200 in the tst if that's the description of the test?

Or, my preference, would be to just change the description to something like:
```suggestion
  test('the works page renders with an accessable title, async () => {
```

Reply:
yeah, I think I'll go with your preference. I was following the test descriptions in the file, but none of those test for 200 codes any more after we changed how the tests worked a while back.

Reply:
Probably worth redescribing the tests then